images/rhel/rhel8/ubi8-toolbox/extra-packages

```diff
@@ -0,0 +1,50 @@
+bzip2
+diffutils
+dnf-plugins-core
+git
+gnupg2-smime
+hostname
+iputils
+less
+lsof
+man-db
+openssh-clients
+pigz
+procps-ng
+rsync
+sudo
+time
+unzip
+vte-profile
+wget
+xz
+zip
+python3
+alternatives
+avahi-libs
+coreutils-common
+dejavu-sans-fonts
+jansson
+libICE
+libSM
+libX11
+libX11-common
+libXau
+libXext
+libXmu
+libXt
+libevent
+libgomp
+libpkgconf
+libxcb
+nano
+pkgconf
+pkgconf-m4
+pkgconf-pkg-config
+python3
+python3-distro
+python3-setuptools
```

Comment:
I don't understand the Python stack in RHEL 8 to know whether or what we should include. :)

I see `python3` listed twice, though. Maybe an alphabetical ordering would make it easier to spot such things?

Reply:
Sorted packages in the file and removed the duplication of python3. Basic python3 it is installed by default in RHEL8.4 installations. That's the reason to add it to the list.